packages/docs/src/lang/en/components/Alerts.json

```diff
@@ -1,78 +1,76 @@
-    },
-    "transition": {
-      "header": "### Transition",
-      "desc": "The **transition** prop allows you to apply a transition to the alert which is viewable when the component hides and shows. For more information, you can checkout any of [Vuetify's prebuilt transitions](/styles/transitions#motion) or review how to [create your own](/styles/transitions#create-your-own)."
-    },
-    "type": {
-      "header": "### Type",
-      "desc": "The **type** prop provides 4 default `v-alert` styles: **success**, **info**, **warning**, and **error**. Each of these styles provide a default icon and color. The default colors can be configured globally by customizing [Vuetify's theme](/customization/theme)."
-    },
-    "twitter": {
-      "header": "### Twitter alert",
-      "desc": "By combining **color**, **dismissible**, **border**, **elevation**, **icon**, and **color-border** props you can create stylish custom alerts such as this twitter notification."
-    }
-  },
-  "accessibility": {
-    "desc": "By default, `v-alert` components are assigned the [WAI-ARIA](https://www.w3.org/WAI/standards-guidelines/aria/) role of [**alert**](https://www.w3.org/TR/wai-aria/#alert) which denotes that the alert \"is a live region with important and usually time-sensitive, information.\" When using the **dismissible** prop the close icon will recieve a corresponding `aria-label`. This value can be modified by changing either the **close-label** prop or globally through customizing the [Internationalization](/customization/internationalization)'s default value for the _close_ property."
-  },
-  "props": {
-    "border": "Puts a border on the alert. Accepts **top** | **right** | **bottom** | **left**.",
-    "closeLabel": "Text used for *aria-label* on **dismissible** alerts. Can also be customizing globally in [Internationalization](/customization/internationalization).",
-    "coloredBorder": "Applies the defined **color** to the alert's border.",
-    "dense": "Hides the alert icon and decreases component's height.",
-    "dismissible": "Adds a close icon that can hide the alert.",
-    "icon": "Designates a specific icon.",
-    "outlined": "Makes the background transparent and applies a thin border.",
-    "prominent": "Displays a larger vertically centered icon to draw more attention.",
-    "text": "Applies the defined **color** to text and a low opacity background of the same.",
-    "type": "Specify a **success**, **info**, **warning** or **error** alert. Uses the contextual color and has a pre-defined icon."
-  },
-  "slots": {
-    "append": "Slot for icon at end of alert.",
-    "prepend": "Slot for icon at beginning of alert."
-  },
-  "scopedSlots": {
-    "close": "Slot for icon used in **dismissible** prop."
-  },
-  "functions": {
-    "toggle": "Toggles the alert's active state. Available in the close slot and used as the click action in **dismissible**."
-  }
-}
+{
+  "header": "# Alerts",

```

Comment:
What changed here?

Reply:
Slot and scopeslots merged. Not sure why git decided that everything changed.

Reply:
Did you mess with line endings?

Reply:
Don't think so, unless linter or something changed them or windows is being cranky. My git config is set to leave them as is though.

Reply:
Nah boi you changed that shit to crlf